extractor/src/test/java/org/schabi/newpipe/extractor/services/peertube/PeertubePlaylistExtractorTest.java

```diff
@@ -30,6 +31,7 @@ public void testGetName() throws ParsingException {
         }
 
         @Test
+        @Ignore("Unknown why url is different")
         public void testGetThumbnailUrl() throws ParsingException {
             assertEquals("https://framatube.org/static/thumbnails/playlist-96b0ee2b-a5a7-4794-8769-58d8ccb79ab7.jpg", extractor.getThumbnailUrl());
```

Comment:
Well, maybe the thumbnail was changed, the [url](https://framatube.org/static/thumbnails/playlist-96b0ee2b-a5a7-4794-8769-58d8ccb79ab7.jpg) leads to a 404

Reply:
The actual url from the test also leads to a 404

Reply:
The thumbnail URLs change every time you make the API request. I tried to fix that a few days ago and stopped frustrated after a few tries.

Reply:
I'll adjust the comment tomorrow, just turned off my pc

Reply:
On a side note: that's a user case for mock downloader and the mock only rule

Reply:
done